packages/@okta/vuepress-site/docs/guides/sign-in-to-spa-authjs/main/react/basic-sign-in.md

```diff
@@ -0,0 +1,71 @@
+This section helps you review the simple password-only sign-in use case from the sample app. This use case is outlined in the following sequence diagram with your single-page app as the client:
+
+Insert vans diagram here
+
+### Set up the Okta configuration settings
+
+Review the `src/config.js` file that references the required [configuration settings](#configuration-settings) to initialize your Okta Auth JS instance. The `config.js`file references the values you add to the `testenv` file.
+
+```JavaScript
+const CLIENT_ID = process.env.SPA_CLIENT_ID || process.env.CLIENT_ID || '{clientId}';
+const ISSUER = process.env.ISSUER || 'https://{yourOktaDomain}.com/oauth2/default';
```

Comment:
is the .com necessary for this? or can it just be 'https://{yourOktaDomain}/oauth2/default';

Reply:
Great catch. I'll remove.